File: DSA-IN-JAVA-Practice/Basic_Math/tempCodeRunnerFile.java
import java.util.*;
public class ArmstrongNumber {
   
    void  Armstrong(int n){
         int sum =0;
         int original =n;
         while(n>0){
            int digit=n%10;
            sum=sum +digit*digit*digit;
            n=n/10;
         }
         if(original==sum){
            System.out.print("Armstrong Number");
         }
         else{
            System.out.print("Not a Armstrong Number");
         }
    }
    public static void main(String [] args){
    System.out.print("Enter your number:");
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    ArmstrongNumber obj=new ArmstrongNumber();
    obj.Armstrong(n);
    
    }
}

File: DSA-IN-C++/Function/tempCodeRunnerFile.cpp
#include <iostream>
using namespace std;
int sum(int a, int b){
    int c=a+b;
return c;
}
int main(){
    int x,y;
    cout<<"enter 2 number";
    cin>>x>>y;
    int result=sum(x,y);
    cout<<result;\
    return 0;
}